web/src/components/netflow-table.tsx

```diff
@@ -19,7 +20,7 @@ const NetflowTable: React.FC<{
   const [activeSortDirection, setActiveSortDirection] = React.useState<string>('asc');
 
   //Sort handler
-  const onSort = (event, index, direction) => {
+  const onSort = (event: any, index: number, direction: string) => {
```

Comment:
Using type any trigger eslint warning (there are 10 of this kind).

I did not look at the other ones but this one can be fixed using React.MouseEvent type.